packages/iov-keycontrol/src/keyring-entries/slip10.ts

```diff
@@ -274,12 +285,11 @@ export class Slip10KeyringEntry implements KeyringEntry {
   }
 
   // This throws an exception when private key is missing
-  private async privkeyForIdentity(identity: PublicIdentity): Promise<Ed25519Keypair> {
+  private async privkeyForIdentity(identity: PublicIdentity): Promise<Uint8Array> {
```

Comment:
This is all private stuff, so it is clear enough between where it is created and where it is used.
On one side, we need to `makeKeypair`, the other just pass it. But you can just scroll down a few lines to understand.